gtsam/geometry/Cal3Fisheye.h

```diff
@@ -160,17 +151,21 @@ class GTSAM_EXPORT Cal3Fisheye {
   /// @name Manifold
   /// @{
 
-  /// Given delta vector, update calibration
-  Cal3Fisheye retract(const Vector& d) const;
-
-  /// Given a different calibration, calculate update to obtain it
-  Vector localCoordinates(const Cal3Fisheye& T2) const;
-
   /// Return dimensions of calibration manifold object
-  virtual size_t dim() const { return 9; }
+  virtual size_t dim() const { return dimension; }
```

Comment:
add to base and override if not just `return Dim(); `

Reply:
Each model has a different dimension (though some have common values for `dimension`), so I'll have to pick and choose.

Reply:
Actually, I can just get rid of the subclass `dim()` methods and use your suggestion of `return Dim()`.

Reply:
Just checked with a unit test: we will need to override `dim()`. :disappointed: